### PR TITLE
Revert pandas to work with Python 3.8 and 3.9

### LINKED ISSUE #3532: Unable to install .[tests] for Python 3.8 and Python 3.9
### Before submitting the issue

- [X] I have searched among the existing issues
- [X] I am using a Python virtual environment

### Description of the bug

Unable to install .[tests] for Python 3.8 and Python 3.9

### Steps To Reproduce

1. Clone PyAedt in eg. D:\pyaedt

2. Create virtual environment (for python 3.8)
py -3.8 -m venv D:\VENV_NAME\venv_3.8

3. Activate virtual environment
call D:\VENV_NAME\venv_3.8\Scripts\activate

python -m pip install pip -U
python -m pip install wheel setuptools -U
python -m pip install D:\pyaedt\.[tests]    -------------------------> This step will error for python 3.8 and 3.9

### Which Operating System are you using?

Windows

### Which Python version are you using?

3.8

### Installed packages

python -m pip install pip -U
python -m pip install wheel setuptools -U

### PR DESCRIPTION
**Pull request by myoung301**
pip install pyaedt[full]/[all]/[tests] fails with Python 3.8 and 3.9 because pandas 2.1.0 cannot be found.

Fixes #3532

### CODE REVIEW

**Comment by myoung301** [comment]
Fixes #3532

File: pyproject.toml [hunk 1]
```diff
@@ -114,7 +114,7 @@ all = [
     "numpy==1.21.6; python_version <= '3.9'",
     "numpy==1.25.2; python_version > '3.9'",
     "pandas==1.3.5; python_version <= '3.7'",
-    "pandas==2.1.0; python_version > '3.7'",
+    "pandas==2.0.3; python_version > '3.7'",
```
Comment (MaxJPRey) [inline pyproject.toml:117]:
```suggestion
    "pandas==2.0.3; python_version <= '3.9'",
    "pandas==2.1.0; python_version > '3.9'",
```

File: pyproject.toml [hunk 2]
```diff
@@ -46,7 +46,7 @@ tests = [
     "openpyxl==3.1.2",
     "osmnx",
     "pandas==1.3.5; python_version <= '3.7'",
-    "pandas==2.1.0; python_version > '3.7'",
+    "pandas==2.0.3; python_version > '3.7'",
```
Comment (MaxJPRey) [inline pyproject.toml:49]:
```suggestion
    "pandas==2.0.3; python_version <= '3.9'",
    "pandas==2.1.0; python_version > '3.9'",
```

**Review by MaxJPRey — APPROVED** [review]
LGTM.

**Pull request merged**